config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
Is this file required?
I just removed it here: https://github.com/3scale/porta/pull/3225/files#diff-188abe2a4493506d81577fbbee0b279b889e59f50d509402fbbfb620c8e58780

I think we can bring it back when we migrate from Paperclip to ActiveStorage

Reply:
Ignore me, I can see it is required in Rails 6... OK :upside_down_face: